package.nls.ko.json

```diff
@@ -0,0 +1,28 @@
+{
+	"plantuml.exportCurrent.title": "현재 다이어그램 내보내기",
+	"plantuml.exportDocument.title": "현재 다이어그램 파일 내보내기",
+	"plantuml.exportWorkspace.title": "작업공간의 다이어그램 내보내기",
+	"plantuml.preview.title": "현재 다이어그램 미리보기",
+	"plantuml.URLCurrent.title": "현재 다이어그램에 대한 URL 생성하기",
+	"plantuml.URLDocument.title": "현재 다이어그램 파일에 대한 URL 생성하기",
+	"plantuml.extractSource.title": "그림에서 PlantUML 다이어그램 소스 추출하기",
+	"plantuml.configuration.configTitle": "PlantUML 환경설정",
+	"plantuml.configuration.exportFormat": "내보내기 형식. 매전 내보내기할 때마다 선택하려면 비워두십시오.",
+	"plantuml.configuration.jar": "plantuml.jar의 다른 위치. 통합 jar를 쓰려면 비워두십시오.",
+	"plantuml.configuration.diagramsRoot": "모든 다이어그램 파일들을 저장할 위치 (작업공간의 상대 폴더).",
+	"plantuml.configuration.fileExtensions": "내보내기 찾을 때 사용할 파일 확장자. 특히 작업공간 설정에서, 소스코드 파일의 다이어그램을 내보내기할 때 \".java\" 같은 사용자의 확장를 추가할 수 있습니다.",
+	"plantuml.configuration.exportSubFolder": "호스트 파일과 같은 이름의 폴더에 다이어그램을 내보낼 수 있습니다.",
+	"plantuml.configuration.exportConcurrency": "여러개의 다이어그램을 내보내기 할 때 동시작업개수를 지정합니다.",
+	"plantuml.configuration.exportOutDir": "내보낸 작업공간 다이어그램이 이 디렉터리에 보관됩니다.  (작업공간의 상대 폴더).",
+	"plantuml.configuration.previewAutoUpdate": "미리보기 화면을 자동으로 업데이트할지를 결정합니다.",
+	"plantuml.configuration.previewSnapIndicators": "미리보기 화면에서 스냅을 표시할지를 결정합니다.",
```

Comment:
This is about snap indicator, so I think this would be better. ([about snap indicator](https://github.com/qjebbs/vscode-plantuml/issues/128))
```
미리보기 화면에서 스냅 지시자를 표시할지를 결정합니다.
```

Reply:
Maybe I will regret this but I will follow direct translation as you proposed. We will need to come up with clearer Korean word for this.

Reply:
Right. Hard to choose appropriate translation for "Indicator" 😥